File: chapter12end/java/aircraft-positions/src/main/java/com/thehecklers/aircraftpositions/RSocketRequesterConfig.java
package com.thehecklers.aircraftpositions;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.messaging.rsocket.RSocketRequester;

@Configuration
public class RSocketRequesterConfig {
    @Bean
    RSocketRequester requester(RSocketRequester.Builder builder) {
        return builder.tcp("localhost", 7635);
    }
}

File: chapter12end/kotlin/aircraft-positions/src/main/kotlin/com/thehecklers/aircraftpositions/RSocketRequesterConfig.kt
package com.thehecklers.aircraftpositions

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.messaging.rsocket.RSocketRequester

@Configuration
class RSocketRequesterConfig {
    @Bean
    fun requester(builder: RSocketRequester.Builder): RSocketRequester {
        return builder.tcp("localhost", 7635)
    }
}
